specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-07-02-preview/managedClusters.json

```diff
@@ -6727,6 +6730,78 @@
       "required": [
         "enabled"
       ]
+    },
+    "ManagedClusterWorkloadAutoScalerProfileVerticalPodAutoscaler": {
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Whether to enable VPA. Default value is false.",
+          "default": false
+        },
+        "controlledValues": {
+          "type": "string",
+          "enum": [
+            "RequestsAndLimits",
+            "RequestsOnly"
+          ],
+          "x-ms-enum": {
+            "name": "ControlledValues",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "RequestsAndLimits",
+                "description": "Autoscaler will control resource requests and limits."
+              },
+              {
+                "value": "RequestsOnly",
+                "description": "Autoscaler will control resource requests only."
+              }
+            ]
+          },
+          "default": "RequestsAndLimits",
+          "description": "Controls which resource value autoscaler will change. Default value is RequestsAndLimits."
+        },
+        "updateMode": {
+          "type": "string",
+          "enum": [
+            "Off",
+            "Initial",
+            "Recreate",
+            "Auto"
+          ],
+          "x-ms-enum": {
+            "name": "UpdateMode",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Off",
+                "description": "Autoscaler never changes pod resources but provides recommendation."
```

Comment:
minor: should say: "but provides recommendations" OR "but provides a recommendation"

Reply:
Thanks for the comment! Updated in the latest iteration.